### PR TITLE
Add `files::update_cloud_status()`

### PR DESCRIPTION
**Pull request by DmitrySamoylov**

### CODE REVIEW

File: src/files.rs [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 use uuid::Uuid;
 
 use super::Client;
 use crate::Result;
 
-#[derive(Clone, Copy, Debug, Eq, PartialEq, Deserialize, Serialize)]
+#[derive(Clone, Copy, Debug, Display, Eq, PartialEq, Deserialize, Serialize)]
 #[serde(rename_all = "snake_case")]
+#[strum(serialize_all = "snake_case")]
 pub enum FileCloudStatus {
```
Comment (FSMaxB) [inline src/files.rs:13]: General question: Is there any guideline which types are moved to [lumeo-types](https://github.com/lumeohq/lumeo-types-rs)?
I've noticed that this type is shared between `api-server` and `api-client`, although `api-server` derives the `sqlx::Type` trait on it, which would require `sqlx` as dependency if the type is shared across repositories.

File: src/files.rs [hunk 2]
```diff
@@ -91,6 +93,18 @@ impl Client {
         self.put(&format!("/v1/apps/{}/files/{}", self.application_id()?, id), file_data).await
     }
 
+    pub async fn update_cloud_status(
+        &self,
+        id: Uuid,
+        cloud_status: &FileCloudStatus,
+    ) -> Result<()> {
+        self.put_text(
+            &format!("/v1/apps/{}/files/{}/cloud_status", self.application_id()?, id),
```
Comment (FSMaxB) [inline src/files.rs:102]: This API endpoint is only available once https://github.com/lumeohq/api-server/pull/79 is merged (and deployed). How are these cross-project dependencies tracked? Via Shortcut stories?

**Review by FSMaxB — APPROVED** [review]

File: src/files.rs (hunk 1 above)
Comment (jplatte) [reply src/files.rs:13]: There's no written policy. I've been thinking about the idea of a crate for api types for a while, but it's not been a high priority.

File: src/files.rs (hunk 2 above)
Comment (DmitrySamoylov) [reply src/files.rs:102]: We just do releases in steps and make sure that everything is compatible at staging and only then release to prod. There's also a page at [Notion](https://www.notion.so/lumeo/Publishing-Releases-ec43b491e14644689636aa5b2c21add2) about the process. It is not enforced in any way, always need to remember that something is incompatible :shrug:

Comment (FSMaxB) [reply src/files.rs:102]: Thanks for explaining!

File: src/files.rs (hunk 1 above)
Comment (DmitrySamoylov) [reply src/files.rs:13]: No guideline yet. This repo contains only REST API calls and types for now. Everything else is living in lumeo-types-rs. 

Later we can move more types here, for example, if we decide to include MQTT comms into this repo.

**Review by jplatte — APPROVED** [review]

**Pull request merged**